config/locales/server.en.yml

```diff
@@ -93,6 +93,16 @@ en:
         markdown_table: Generate Markdown table
 
     ai_bot:
+      personas:
+        general:
+          name: Default
```

Comment:
With the dropdown, having "Default" showing at first seems a little uninspiring and doesn't tell that this is for personas (is personas an end-user term?):

![image](https://github.com/discourse/discourse-ai/assets/920448/fff5bda9-8d9f-4327-9e1f-9216dfe1f75b)

Maybe the default one could be called "Flexible" or "Flexible Persona"? Or maybe "Versatile"?

Reply:
Yeah tricky, not sure ... will try Forum Helper for now ... though Jammy wants it to be Bert and Lilly wants it to be Lola...